full-stack-tests/core/README.md

```diff
@@ -22,14 +22,14 @@ To run the integration tests, there is an initial configuration step required to
 
     ```sh
     # User registered with iTwin Platform
-    imjs_test_regular_user_name=
-    imjs_test_regular_user_password=
+    IMJS_TEST_REGULAR_USER_NAME=
+    IMJS_TEST_REGULAR_USER_PASSWORD=
 
     # Register a new SPA client at https://developer.bentley.com/my-apps.
     # Ensure the client is configured for  the redirect uri and scopes below.
-    imjs_oidc_browser_test_client_id=
-    imjs_oidc_browser_test_redirect_uri="http://localhost:3000/signin-callback"
-    imjs_oidc_browser_test_scopes="openid email profile organization imodelhub context-registry-service:read-only product-settings-service"
+    IMJS_OIDC_BROWSER_TEST_CLIENT_ID=

```

Comment:
Lets catch up tomorrow, I want to ensure this isn't going to break a bunch of our internal testing setup.